conf/defaults.ini

```diff
@@ -2191,3 +2191,15 @@ fail_tests_on_console = true
 # Whether to enable betterer eslint rules for local development
 # Useful if you want to always see betterer rules that we're trying to fix so they're more prevalent
 betterer_eslint_rules = false
+
+#################################### Plugin API Restrictions ##########################################
+# Configure which plugins can access specific restricted APIs.
+# Use plugin IDs or regex patterns. Whitelist takes precedence over blacklist.
+
+[plugins.restricted_apis_whitelist]
+# Example: Allow specific plugins to access an API
+# addPanel = "myorg-admin-app, grafana-enterprise-.*"
+
+[plugins.restricted_apis_blacklist]
```

Comment:
```suggestion
[plugins.restricted_apis_blocklist]
```

Reply:
Or `denylist`?

Reply:
Updated in https://github.com/grafana/grafana/pull/107967/commits/6833830c5033ece7a376f9122dd35b307c528a54. (`denylist` is good as well, but in the end I went with `blocklist`).